docs/guide/localization/static.md

```diff
@@ -0,0 +1,199 @@
+# Static content localization
+
+This section covers localization of strings throughout template files and Apostrophe user interface. These strings are hard-coded, not editable by logged-in users, and thus are not stored in the database. Instead, this localization is stored in JSON files associated with the various locales.
+
+## Registering strings to localize
+
+There are often good reasons to hard-code text in templates even if we operate in multiple languages. For example, if our website has a blog with related articles featured at the bottom of each post using a relationship field. The heading "Related articles" belongs in the show page template. We don't want to make content editors translate that each time and we want to avoid creating any custom global fields for a text string that is not likely to change.
+
+That section of our show page template might look like this:
+
+<AposCodeBlock>
+  ```django
+    {# More article template stuff above ⤴ #}
+    <section>
+      <h2>Related articles</h2> {# 👈 We need to register this. #}
+      <ul>
+        {% for post in data.piece._related %}
+          <li><a href="{{ post._url }}">{{ post.title }}</a></li>
+        {% endfor %}
+      </ul>
+    </section>
+  ```
+  <template v-slot:caption>
+    /modules/article-page/views/show.html
+  </template>
+</AposCodeBlock>
+
+Registering that string is as easy as wrapping it in a template method: `__t()`. Make sure the string passed to the method is in quotes and any matching quotes in the string are escaped (e.g., `__t('Gritty\'s friends')`).
```

Comment:
template helper, or template helper function when it's not relaxing with friends. It's not being invoked with method syntax, so I think that language is confusing.